.circleci/config.yml

```diff
@@ -8,6 +8,7 @@ jobs:
   build:
     executor:
       name: node/node
+      node-version: 10.24
```

Comment:
Since we support from node 10 upwards, make sure we use that version for testing :) the default otherwise is 14 as of today